### DISCUSSION TITLE
How force set param in every use of "new Class(...., $param)"

### DISCUSSION THREAD

**Discussion by fernirek** [rectorphp/rector#8233] [Q&A] [answered]
I'm having trouble creating my own rule. The problem I wanted to solve 
was that I wanted to force the 'secure' parameter to be true in every use of "new Cookie(...)" in the code. I can't find the right getNodeTypes, and how to get to the right constructor in recator method.

e,g,
new Cookie('test', 'abcd', 1000, '/', null, **false**, false) => new Cookie('test', 'abcd', 1000, '/', null, **true**, false)

please help, or give me some hints.

**Comment by TomasVotruba** [comment, marked as answer]
That would be `New_` node. Then `$node->getArgs()` called on it.

This node documentation might give you more hints: https://github.com/rectorphp/php-parser-nodes-docs#phpparsernodeexprnew_
